config.js

```diff
@@ -133,16 +133,22 @@ exports.updateFromArgv = function() {
         .default('timecontrols', 'fischer,byoyomi,simple,canadian')
         .describe('timecontrolsranked', 'Time control(s) to accept for ranked games')
         .describe('timecontrolsunranked', 'Time control(s) to accept for unranked games')
-        //         B2) GENERIC GENERAL/RANKED/UNRANKED ARGUMENTS : :
+        //         B2) GENERIC GENERAL/RANKED/UNRANKED ARGUMENTS:
         .describe('proonly', 'For all matches, only accept those from professionals')
         .describe('proonlyranked', 'For ranked matches, only accept those from professionals')
         .describe('proonlyunranked', 'For unranked matches, only accept those from professionals')
+                /* note: - nopause allows to disable pauses DURING games, (game.js), but
```

Comment:
This might be more clear as "nopause disables pausing DURING games...".

Reply:
fixed :)